libs/tunables/tunables_shell.py

```diff
@@ -0,0 +1,113 @@
+import cmd
+import yaml
+
+from ast import literal_eval
+from datetime import date
+
+
+# import database
+
+
+class tunables(cmd.Cmd):
+    print(
+        " \n This is the Tunables Parameters program. \n Type in ? to see the list of commands"
+    )
+    prompt = "(tunables) "
+    file = None
+
+    # Commands
+
+    def do_get(self, arg):
+        # database.send(0, arg[0])
+        print(search_yaml(arg))
+
+    def do_set(self, arg):
+        print(parse(arg)[0])
+
+        # database.send(0, arg[0], arg[1])
+        write_yaml(parse(arg))
+        print(search_yaml(parse(arg)[0]))
+
+    def do_list(self, arg):
+        list_yaml()
+
+
+def parse(arg):
+    return arg.split(" ")
+
+
+def search_yaml(arg):
+
+    """Finds information of 1 parameter"""
+    with open("tunables.yml", "r") as file:
+        try:
+            data = yaml.safe_load(file)
+
+            for i in range(len(data)):
+                for j in range(len(data[i]["params"])):
```

Comment:
Currently the way the yaml file is formatted we don't use a dictionary so it's hard to search by keyword. Also since each board will have max 16~ tunable parameters, I think the for loop is a heuristic solution for now.